meinberlin/apps/kiezradar/models.py

```diff
@@ -50,18 +61,8 @@ class Meta:
 
 
 class ProjectType(models.Model):
-    PARTICIPATION_INFORMATION = 0
-    PARTICIPATION_CONSULTATION = 1
-    PARTICIPATION_COOPERATION = 2
-    PARTICIPATION_DECISION_MAKING = 3
-    PARTICIPATION_CHOICES = (
-        (PARTICIPATION_INFORMATION, _("information (no participation)")),
-        (PARTICIPATION_CONSULTATION, _("consultation")),
-        (PARTICIPATION_COOPERATION, _("cooperation")),
-        (PARTICIPATION_DECISION_MAKING, _("decision-making")),
-    )
     participation = models.SmallIntegerField(
-        choices=PARTICIPATION_CHOICES,
+        choices=Plan.PARTICIPATION_CHOICES,
```

Comment:
I'd suggest to do the other way around (import these from here in the `plan.models`), since participation is related to project type, than just the plan. But keep the order as it is in the `plan.PARTICIPATION_CHOICES`.
@vellip @sevfurneaux is this what you fixed in this [PR](https://github.com/liqd/a4-meinberlin/pull/6080/)?